webapp/components/post_view/components/post_message_view.jsx

```diff
@@ -55,6 +58,10 @@ export default class PostMessageView extends React.Component {
             return true;
         }
 
+        if (!Utils.areObjectsEqual(nextProps.liteUsernameMap, this.props.liteUsernameMap)) {
+            return true;
+        }
```

Comment:
This makes me nervous, posts re-render a lot and this could potentially cause some slow down. I think it might be better to just use the regular usernameMap and don't even check it here. The updates might be a bit late since we'll need to wait until this post remounts but it's a lot better on the performance when re-rendering all the time